python/pyspark/sql/functions.py

```diff
@@ -189,15 +210,15 @@ def _():
 }
 
 for _name, _doc in _functions.items():
-    globals()[_name] = since(1.3)(_create_function(_name, _doc))
+    globals()[_name] = since(1.3)(ignore_unicode_prefix(_create_function(_name, _doc)))
```

Comment:
`ignore_unicode_prefix` is necessary to get the tests to pass both python 2 and python 3. I had problems without it on my previous documentation PR: https://github.com/apache/spark/pull/17469#issuecomment-295896057